### PR TITLE
Clientcore integration

### PR DESCRIPTION
**Pull request by ObsessiveOrange**
Editor now can send data to server. Some items are hard coded (ProjectID, userID, FileID, etc). However, these are blocked by GUI.

### CODE REVIEW

File: src/main/java/cceclipseplugin/core/PluginManager.java [hunk 1]
```diff
@@ -49,30 +61,124 @@ public static PluginManager getInstance() {
...
+								if (response.getStatus() != 200) {
+									throw new IllegalStateException("Failed to log in");
+								}
+
+								getWSManager().setAuthInfo(StringConstants.PREFERENCES_USERNAME,
+										((UserLoginResponse) response.getData()).getToken());
+
+								try {
+									wsManager.sendRequest(new ProjectSubscribeRequest(StringConstants.PROJ_ID)
+											.getRequest(null, null));
```
Comment (loganga) [inline src/main/java/cceclipseplugin/core/PluginManager.java:155]: Shouldn't we have response and error handlers here? There's no way of knowing if it failed.

File: src/main/java/cceclipseplugin/core/PluginManager.java [hunk 2]
```diff
@@ -49,30 +61,124 @@ public static PluginManager getInstance() {
...
+								// ("package testPkg1;\n" + "\n" + "public class
+								// TestClass1 {\n"
+								// + "\n" + "}").getBytes());
+								// try {
+								// getWSManager().sendRequest(data.getRequest());
+								// } catch (ConnectException e) {
+								// // TODO Auto-generated catch block
+								// e.printStackTrace();
+								// }
+							}, null);
```
Comment (loganga) [inline src/main/java/cceclipseplugin/core/PluginManager.java:174]: Same here, no error handler

Comment (loganga) [reply src/main/java/cceclipseplugin/core/PluginManager.java:174]: Also on line 180

File: src/main/java/cceclipseplugin/core/PluginManager.java (hunk 1 above)
Comment (ObsessiveOrange) [reply src/main/java/cceclipseplugin/core/PluginManager.java:155]: Yep. these were there to allow it to function while waiting for our GUI to be functional. This is a temporary measure that should be removed once GUI is integrated.

All of the hardcoding in this should disappear once we can get a GUI, and pass the values dynamically.

File: src/main/java/cceclipseplugin/core/PluginManager.java (hunk 2 above)
Comment (ObsessiveOrange) [reply src/main/java/cceclipseplugin/core/PluginManager.java:174]: Same as above

File: src/main/java/cceclipseplugin/core/PluginManager.java (hunk 1 above)
Comment (loganga) [reply src/main/java/cceclipseplugin/core/PluginManager.java:155]: Ok, just making sure. I've got a couple of classes that are just generic response and error handlers that open up dialogs, so it should be a really simple change once those are merged.

**Comment by ObsessiveOrange** [comment]
If everything looks good, can you squash and merge this into test?

**Pull request merged**